packages/spacecat-shared-data-access/src/models/configuration.js

```diff
@@ -17,14 +17,145 @@ const Configuration = (data = {}) => {
   self.getJobs = () => self.jobs;
   self.getVersion = () => self.version;
   self.getQueues = () => self.queues;
+  self.getHandlers = () => self.handlers;
+  self.getHandler = (type) => self.handlers[type];
+  self.isHandlerTypeEnabledForSite = (type, site) => {
+    if (self.handlers[type]?.enabled) {
+      if (self.handlers[type].enabled.sites.includes(site.getId())) {
+        return true;
+      } else {
+        const orgId = site.getId();
+        return self.handlers[type].enabled.orgs.includes(orgId);
+      }
+    }
+    if (self.handlers[type]?.disabled) {
+      if (self.handlers[type].disabled.sites.includes(site.getId())) {
+        return false;
+      } else {
+        const orgId = site.getId();
+        return !self.handlers[type].disabled.orgs.includes(orgId);
+      }
+    }
+    return self.handlers[type].enabledByDefault;
+  };
+
+  self.isHandlerTypeEnabledForOrg = (type, org) => {
+    if (self.handlers[type]?.enabled) {
+      return self.handlers[type].enabled.orgs.includes(org.getId());
+    } else if (self.handlers[type]?.disabled) {
+      return !self.handlers[type].disabled.orgs.includes(org.getId());
+    }
+    return self.handlers[type].enabledByDefault;
+  };
```

Comment:
while slightly longer, this would be more readable:
```suggestion
self.isHandlerTypeEnabledForOrg = (type, org) => {
  const handler = self.handlers[type];
  if (!handler) return false;

  const orgId = org.getId();

  if (handler.enabled) {
    return handler.enabled.orgs.includes(orgId);
  }

  if (handler.disabled) {
    return !handler.disabled.orgs.includes(orgId);
  }

  return handler.enabledByDefault;
};
```